zstacklib/zstacklib/utils/iproute.py

```
@@ -671,14 +735,16 @@ def add_namespace(namespace):
     try:
         pyroute2.netns.create(namespace)
     except OSError as e:
-        if e.errno == os.errno.EEXIST: # namespace already exists
+        if e.errno == os.errno.EEXIST:  # namespace already exists
             raise NamespaceAlreadyExists(namespace)
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)